salt/transport/zeromq.py

```diff
@@ -1373,11 +1373,16 @@ def timeout_message(self, message):
                 future.set_exception(SaltReqTimeoutError("Message timed out"))
 
     def send(
-        self, message, timeout=None, tries=3, future=None, callback=None, raw=False
+        self, message, timeout=None, tries=None, future=None, callback=None, raw=False
     ):
         """
         Return a future which will be completed when the message has a response
         """
+        # Retrieve auth_tries from configuration if None is passed
+        # Otherwise, use 3
+        if tries is None:
+            tries = self.opts.get("auth_tries", 3)
```

Comment:
```suggestion
            tries = self.opts.get("auth_tries") or 3
```
If `auth_tries` is set to None, we would never default to 3.